apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts

```diff
@@ -0,0 +1,54 @@
+/* istanbul ignore file */
+import { RestTeam, RestUser } from '@asap-hub/squidex';
+import { Migration } from '../handlers/webhooks/webhook-run-migrations';
+import { applyToAllItemsInCollection } from '../utils/migrations';
+
+export default class MoveRepurposedFields extends Migration {
+  up = async (): Promise<void> => {
+    await migrateUserFields();
+    await migrateTeamFields();
+  };
+  down = async (): Promise<void> => {
+    /* Ignore  */
+  };
+}
+async function migrateUserFields() {
+  await applyToAllItemsInCollection<RestUser>(
+    'users',
+    async (
+      {
```

Comment:
personally i find destructuring such complex data structures a little hard to read, would you be better off using a simple variable here and the referencing the props via dot notation?

Reply:
Done